src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServerWrapper.cs

```diff
@@ -53,6 +53,9 @@ private RazorLanguageServerWrapper(RazorLanguageServer innerServer)
             TraceSource = traceSource
         };
 
+        // Get more information about exceptions that occur during RPC method invocations.
+        jsonRpc.ExceptionStrategy = ExceptionProcessing.ISerializable;
```

Comment:
Nit: Consider moving into the `CreateJsonRpc` method, just in case something else calls it at some point.

Reply:
I like it